pyomo/contrib/mindtpy/tests/from_proposal.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
 """
 See David Bernal PhD proposal example.
```

Comment:
Add this link: https://www.researchgate.net/project/Convex-MINLP/update/5c7eb2ee3843b034242e9e4a